nulib/meadow
# Add replace fileset feature in the Preservation tab for a Work

# Summary 
Adds ability to replace a file set in the Meadow UI.  The only pieces of information which are updatable will be `Label` and `Description`.



# Specific Changes in this PR
- Adds "Replace fileset" option in the Actions dropdown menu on a Work > Preservation tab row
- Adds a new modal window for Replace File Set

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
1. Go to a Work with a Fileset you can change with no collateral damage
2. Scroll down to the tabs, Preservation tab and from the Actions dropdown select "Replace fileset"
3. Select a new file
4. Note previous `Label` and `Description` values should be default values
5. Submit the form
6. Notice the changes in the Structure and Preservation tabs for the Fileset (note this may take a few minutes to populate)
 

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
